### ISSUE TITLE
Iteration of Tiled Gallery settings

### ISSUE THREAD

**Issue by MichaelArestad** [Automattic/jetpack#7576]
Currently Tiled Gallery is connected to Photon in settings. It looks like this:
![image](https://user-images.githubusercontent.com/1123119/28887080-0e559186-7779-11e7-85be-bc2b9acfb562.png)

After some discussion with @rickybanister, we think it would be a good idea to move it and provide access to a suboption that is missing from Jetpack Settings, yet available in wp-admin.

It would look like this:
![image](https://user-images.githubusercontent.com/1123119/28887439-72f48dd0-777a-11e7-9ee4-35bda61c8e74.png)

We would also need to remember to update the search keywords and one-way updates for tiled galleries.

We should also mirror this change in Calypso. CC @tyxla @oskosk

**Comment by rickybanister** [comment]
That's so much more clear.

Also cc @dereksmart

**Comment by MichaelArestad** [comment]
Benefits:
* Exposes tiled galleries a bit more
* allows the ability to set all galleries as a tiled mosaic

**Comment by danjjohnson** [comment]
Maybe this should be a separate issue, but...

![image](https://user-images.githubusercontent.com/1990037/28909837-15b33a9c-7822-11e7-989d-8e99cd04eaf9.png)

The second line says

> Show photo metadata (Exif) in carousel, when available

But the first line doesn't mention a carousel, so it's kinda confusing.

Should we change the first line to:

> Display images and galleries in a gorgeous, full-screen carousel

**Comment by rickybanister** [comment]
Good catch @danjjohnson. We should also remove the word 'gorgeous'—that editorialization didn't make it into the wordpress.com version of these settings and doesn't add much. It's merely our opinion that it's gorgeous.

**Comment by tyxla** [comment]
This could still be beneficial.

**Comment by jeherve** [comment]
Related: #10753
